ReactAndroid/build.gradle

```diff
@@ -210,14 +210,9 @@ def findNdkBuildFullPath() {
         def ndkDir = property("ndk.path")
         return new File(ndkDir, getNdkBuildName()).getAbsolutePath()
     }
-    if (System.getenv("ANDROID_NDK") != null) {
```

Comment:
Still keep the getenv logic ?

Reply:
Android plugin is reading it and making available via android.ndkDirectory, so I removed the duplication.